schema_based_on_diagram.sql

```diff
@@ -0,0 +1,61 @@
+-- create patients table
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name TEXT NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+-- create medical_histories table
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP NOT NULL,
+  status TEXT NOT NULL,
```

Comment:
- [ ] Kindly ensure you use the same data type shown in the ERD. `Status` should be `varchar` and not `text`. Deviating from the ERD can introduce inconsistencies and make it harder to understand and maintain the database structure.